### DISCUSSION TITLE
Alcance de la solución

### DISCUSSION THREAD

**Discussion by pakosanchez27** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#87] [Q&A] [answered]
Hola compañeros alguien que me pueda explicar que se pone en alcance de la solución

**Comment by Marco01nD** [comment, marked as answer]
Hola, se refiere a que finalmente esta implementación (cambio, modificación o actualización del proyecto) hasta que punto te va a llevar.
Ejemplo: Al generar este formulario lo que se pretende es que va a cubrir algunas demandas que en su mayoría no requieran gran cantidad de información adicional o de documentos, o que en su defecto sea un proceso que no dure mas de unas semanas, para demandas que excedan cierto limite, el llenar el formulario se va a quedar corto de información, prácticamente se pretende abarcar aquellas demandas mas comunes que llegan de una manera mas recurrente al despacho.


Espero te pueda ayudar mi respuesta. Saludos.

**Reply by pakosanchez27** [reply]
Muchas Gracias!!
